### PR TITLE
Upgrade isort==5.9.1

### PR DESCRIPTION
**Pull request by cclauss**
Fixes #1382
Fixes #1384
Fixes #1387
https://pypi.org/project/isort

### CODE REVIEW

**Comment by RobbeSneyders** [comment]
Thx @cclauss.

There seem to be two reasons isort is failing:
- Isort thinks `aiohttp` is local in the tests because of the `tests/aiohttp` directory. We can pass `aiohttp` as a `--thirdparty` to fix this.
- You made some changes which treat `conftest` as third party. I think we need to revert those changes.

**Comment by cclauss** [comment]
Bullet 1: I did try `isort --thirdparty connexion,aiohttp ./tests` and that might have gotten us closer but still not yet passing.

Bullet 2: The only [PR that I landed](https://github.com/zalando/connexion/pulls?q=is%3Apr+is%3Aclosed+author%3Acclauss) that modified `./tests` was #1383 but that did not modify import ordering.  There seem to be [isort issues with conftest.py](https://github.com/PyCQA/isort/search?q=conftest&type=issues).

File: tests/decorators/test_validation.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
 from unittest.mock import MagicMock
 
 import pytest
+from jsonschema import ValidationError
+
 from connexion.apis.flask_api import FlaskApi
```
Comment (RobbeSneyders) [inline tests/decorators/test_validation.py:6]: This change doesn't treat connexion as third party anymore.

**Review by RobbeSneyders — COMMENTED** [review]
I meant the changes to `tests` in this PR. Added an in-line comment with one example.

**Comment by cclauss** [comment]
Now I understand...

`tox` does `cd tests` before it runs `isort` which produces different results that `isort tests`.  Thanks for the guidance.

**Review by RobbeSneyders — APPROVED** [review]
Ah, that explains it. Thanks a lot for this and your other PRs!

**Pull request merged**